docs/config/index.md

```diff
@@ -82,6 +82,25 @@ export default defineConfig(async ({ command, mode }) => {
 })
 ```
 
+
+## Hot reload monorepos packages
+
+To have your library packages hot reload in local dev, add an alias to resolve to your package.
+
+Example:
+```js
+{
+  resolve: {
+    alias: [
+      {
+        find: /^@pkg\/(utils|components)$/,
+        replacement: join(__dirname, "packages/$1/src/index.ts")
+      }
+    ]
+  }
+```
```

Comment:
I believe Vite does HMR for monorepo pacakges by default without this setting? 🤔 As long as it's linked locally, e.g. `"some-lib": "workspace:*"` in `package.json`, it should work by default. Unless it's the `file:` protocol it won't work as there's the `node_modules` keyword in the resolved package path.